File: my_stack/my_turtle_controller/scripts/controller.py
#!/usr/bin/env python
import roslib
roslib.load_manifest('my_turtle_controller'); import rospy

from turtlesim.msg import *

vel = Velocity()
vel.linear = 1.
vel.angular = 1.

pub = rospy.Publisher('/turtle1/command_velocity', Velocity)
rospy.init_node('controller')

while not rospy.is_shutdown():
    pub.publish(vel)
    rospy.sleep(1.0)

File: my_stack/my_turtle_controller/src/controller.cpp
#include <ros/ros.h>

#include <turtlesim/Velocity.h>

int
main (int argc, char* argv[])
{
  ros::init (argc, argv, "my_controller");

  ros::NodeHandle nodeHandle;

  ros::Rate rate (1000);
  
  ros::Publisher pub = nodeHandle.advertise<turtlesim::Velocity>
    ("/turtle1/command_velocity", 1);

  turtlesim::Velocity vel;
  
  while (ros::ok ())
    {
      vel.linear = 1.;
      vel.angular = 1.;
      pub.publish (vel);
      ros::spinOnce ();
      rate.sleep ();
    }
  
  return 0;
}
